impact/workflows/CreateProject.md

```diff
@@ -0,0 +1,390 @@
+# Impact API - Create Project Workflow
+- This document is a supplement to the [Impact ReadMe](https://github.com/Implan-Group/api/blob/main/impact/readme.md)
+- Further detail on additional workflow topics can be found on [support.implan.com](https://support.implan.com/hc/en-us)
+- Projects are the top level of organization for an analysis and contain the specifics of the Aggregation Scheme, Household Set, MRIO status, and folder placement.
+- This document describes the basic IMPLAN ImpactAPI workflow to create a Project, add Events, and link Events and Regions in Groups
+
+### Notes:
+- All API Endpoints require a valid JWT Bearer Token ([JWT.IO](https://jwt.io/))
+  - Please see the [authentication](https://github.com/Implan-Group/api/blob/main/impact/readme.md#authentication---retrieving-bearer-access-token) section of the primary Readme to review authentication steps
+- Variables for endpoints will appear inside of double-braces, those must be replaced with valid values before execution
+  - e.g. `{{api_domain}}` should be replaced with `https://api.implan.com/`
+  - See the [Development Variables](https://github.com/Implan-Group/api/blob/main/impact/readme.md#development-variables) section of the Readme for more information
+- Endpoints that combine IMPLAN's economic data with other data are limited to only those datasets to which those other data are available. (This primarily effects Canada + International customers)
+
```

Comment:
@TRJ-Implan can we add an overview?

Reply:
Are you suggesting a more detailed overview of the Authentication process?

Reply:
I believe this is in reference to having an overview section with a brief explanation of the purpose for a `Project`. Much like you have in your CombinedRegion workflow.

Reply:
Correct @MT737 .  @TRJ-Implan looking for the same general structure as the other md files.